Authentication/Login/View/LoginView.swift

```diff
@@ -129,132 +115,183 @@ public final class LoginView: UIView, LoginViewType, NibLoadable {
-            } else {
-                passwordErrorsHeightConstraint.constant = 0
-                passwordErrorsHeightConstraint.active = true
-            }
+    
+    private func emailTextFieldSelectedWasSet() {
+        if emailTextFieldSelected {
+            emailTextFieldViewOutlet.layer.borderColor = delegate.colorPalette.textfieldsSelected.CGColor
+            emailErrorsHeightConstraint.constant = 0
+            emailErrorsHeightConstraint.active = true
+        } else {
+            let valid = emailTextFieldValid
+            emailTextFieldValid = valid
         }
     }
     
-    public var passwordTextFieldSelected = false {
-        didSet {
-            if passwordTextFieldSelected {
-                passwordTextFieldAndButtonViewOutlet.layer.borderColor = delegate.colourPalette.textfieldsSelected.CGColor
+    
+    private func passwordTextFieldValidWasSet() {
+        if !passwordTextFieldSelected {
+            let color: CGColor
+            if passwordTextFieldValid {
+                color = delegate.colorPalette.textfieldsNormal.CGColor
                 passwordErrorsHeightConstraint.constant = 0
                 passwordErrorsHeightConstraint.active = true
             } else {
-                let valid = passwordTextFieldValid
-                passwordTextFieldValid = valid
+                color = delegate.colorPalette.textfieldsError.CGColor
+                passwordErrorsHeightConstraint.active = false
             }
+            passwordTextFieldAndButtonViewOutlet.layer.borderColor = color
+        } else {
+            passwordErrorsHeightConstraint.constant = 0
+            passwordErrorsHeightConstraint.active = true
         }
     }
     
-    public var logInButtonEnabled: Bool = true {
-        didSet {
-            let color: UIColor
-            if logInButtonEnabled {
-                color = delegate.colourPalette.logInButtonEnabled
-            } else {
-                color =  delegate.colourPalette.logInButtonDisabled
-            }
-            logInButton.backgroundColor = color
-        }
-    }
     
-    public var logInButtonPressed = false {
-        didSet {
-            let color: UIColor
-            if logInButtonPressed {
-                color =  delegate.colourPalette.logInButtonExecuted
-            } else {
-                color =  delegate.colourPalette.logInButtonEnabled
-            }
-            logInButton.backgroundColor = color
-            emailErrorsHeightConstraint.constant = 0
-            emailErrorsHeightConstraint.active = true
+    private func passwordTextFieldSelectedWasSet() {
+        if passwordTextFieldSelected {
+            passwordTextFieldAndButtonViewOutlet.layer.borderColor = delegate.colorPalette.textfieldsSelected.CGColor
             passwordErrorsHeightConstraint.constant = 0
             passwordErrorsHeightConstraint.active = true
+        } else {
+            let valid = passwordTextFieldValid
+            passwordTextFieldValid = valid
```

Comment:
It's to trigger the effect that is already defined in the didSet of the `passwordTextFieldValid ` property. And if done in one line swiftlint complains, if I'm not mistaken.